cmd/serve.go

```diff
@@ -101,6 +104,7 @@ func init() {
 	serveCmd.Flags().StringVar(&queueName, "queue-name", "", "the name of the SQS queue to consume lifecycle hooks from")
 	serveCmd.Flags().StringVar(&kubectlLocalPath, "kubectl-path", "/usr/local/bin/kubectl", "the path to kubectl binary")
 	serveCmd.Flags().StringVar(&logLevel, "log-level", "info", "the logging level (info, warning, debug)")
+	serveCmd.Flags().Int64Var(&maxDrainConcurrency, "max-drain-concurrency", 32, "maximum number of node drains to process in parallel")
```

Comment:
You might just want the unsigned int `uint` instead, then you just convert to int64 for the `semaphore.NewWeighted(int64(maxDrainConcurrency))`

Reply:
What is the difference with current? `uint` is simply not accepting negative numbers, it would also be possible to use a standard int and convert to int64

Reply:
I guess my point is, if we know we only use this variable as an `int64` why do we need an extra conversion?